openlcsd/flow/tasks.py

```diff
@@ -849,7 +849,7 @@ def save_package_data(context, engine):
             'task_id': context.get('task_id')
         }
         json.dump(file_content, destination, cls=DateEncoder)
-    resp = cli.post(url, data={"file_path": tmp_file_path})
+    resp = cli.post(url, data={"file_path": tmp_file_path}, timeout=420)
```

Comment:
Any clue 420 used and how this update could solve the timeout issue?

Reply:
Just try to reduce the frequency of timeout problem, not ensure 100% fixed. This is not a 100% reproduce error. What i do in this PR is try to reduce the frequency of this error.